pkg/util/util.go

```diff
@@ -10,12 +10,15 @@ import (
 	"time"
 )
 
-var (
-	maskPasswordRegex1         = regexp.MustCompile(`password=(\s*?)'([^']+?)'`)
+const (
+	PromNamespace              = "ts_prom"
 	maskPasswordReplaceString1 = "password=$1'****'"
-	maskPasswordRegex2         = regexp.MustCompile(`password:(\s*?)([^\s]+?)( |$)`)
 	maskPasswordReplaceString2 = "password:$1****$3"
-	PromNamespace              = "ts_prom"
+)
+
+var (
+	maskPasswordRegex1 = regexp.MustCompile(`password=(\s*?)'([^']+?)'`)
+	maskPasswordRegex2 = regexp.MustCompile(`password:(\s*?)([^\s]+?)( |$)`)
```

Comment:
I don't think I did this. Maybe the rebasing messed this. Should I undo?

Reply:
Do you have any automated tools that fixed this up? Its better like this anyways.